hack/artifacts.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+
+mkdir -p /tmp/artifacts
+
+export KUBECONFIG=$GOPATH/src/github.com/openshift/openshift-azure/_data/_out/admin.kubeconfig
```

Comment:
can we rely on cwd here - do we need `$GOPATH/src/github.com/openshift/openshift-azure/` hardcoded?